Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN cd /source && \
     mv ./*.zip / && \
     cd / && \
     unzip /*.zip -d / && \
-    wget http://central.maven.org/maven2/mysql/mysql-connector-java/5.1.39/mysql-connector-java-5.1.39.jar && \
+    wget https://central.maven.org/maven2/mysql/mysql-connector-java/5.1.39/mysql-connector-java-5.1.39.jar && \
```

Comment:
Same thing here:

this should be repo1.maven.org instead of central.maven.org because of this: SSL: no alternative certificate subject name matches target host name 'central.maven.org'